.github/pull_request_template.md

```diff
@@ -34,56 +7,16 @@ ACTION: Delete any content types that don't apply to your pull request
 📝 Documentation
 🛠️ Repo maintenance
 
-## Description
+## PR Description
 
-<!--
-ACTION: Summarise your Pull Request
--->
 
 ## Related Issues or other material
-
-- [ ] This PR relates to one or more issues, detailed below
-
-<!--
-ACTION: Detail any issues this PR relates to, and then check the box.
-
-NOTE: PRs without issues will not be merged.
--->
-
-<!--
-We like to follow [Github's guidance on linking issues to pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
-
-For example having the text: "closes #1234" would connect the current pull
-request to issue 1234.  And when we merge the pull request, Github will
-automatically close the issue.
-
-Be careful not to close parent tracking issues that this PR only completes PART of.
--->
-
-- Related Issue #
-- Closes #
+Closes #
```

Comment:
```suggestion
Related #
Closes #
```

Just really don't want people to close issues they don't mean to when linking related